spec/five_day_weather_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'spec_helper'
+
+describe 'latest five day forecast weather data through name and id' do
+
+  context 'test five day forecast weather data by name' do 
+
+    before(:all) do
+      @current = Weather.new.fivedayWeather
+      @current.retrieve_five_day_weather_name('London')
+    end
+
+    it 'should return a string data type from the cod key' do
```

Comment:
Need to keep your test descriptions consistent, sometimes you have said 'should return a string data type from the cod key' and then 'should retrieve the pressure key as an integer data type'. You are referring the the type at the start in one and then at the end in the other. You don't need to say data type.